sdk/python/feast/usage.py

```diff
@@ -25,76 +25,72 @@
 
 from feast.version import get_version
 
-TELEMETRY_ENDPOINT = (
-    "https://us-central1-kf-feast.cloudfunctions.net/bq_telemetry_logger"
-)
+USAGE_ENDPOINT = "https://us-central1-kf-feast.cloudfunctions.net/bq_telemetry_logger"
 _logger = logging.getLogger(__name__)
 
 
-class Telemetry:
+class Usage:
     def __init__(self):
-        self._telemetry_enabled: bool = False
+        self._usage_enabled: bool = False
         self.check_env_and_configure()
 
     def check_env_and_configure(self):
-        telemetry_enabled = (
-            os.getenv("FEAST_TELEMETRY", default="True") == "True"
+        usage_enabled = (
+            os.getenv("FEAST_USAGE", default="True") == "True"
         )  # written this way to turn the env var string into a boolean
 
         # Check if it changed
-        if telemetry_enabled != self._telemetry_enabled:
-            self._telemetry_enabled = telemetry_enabled
+        if usage_enabled != self._usage_enabled:
+            self._usage_enabled = usage_enabled
 
-            if self._telemetry_enabled:
+            if self._usage_enabled:
                 try:
                     feast_home_dir = join(expanduser("~"), ".feast")
                     Path(feast_home_dir).mkdir(exist_ok=True)
-                    telemetry_filepath = join(feast_home_dir, "telemetry")
+                    usage_filepath = join(feast_home_dir, "usage")
 
-                    self._is_test = (
-                        os.getenv("FEAST_IS_TELEMETRY_TEST", "False") == "True"
-                    )
-                    self._telemetry_counter = {"get_online_features": 0}
+                    self._is_test = os.getenv("FEAST_IS_USAGE_TEST", "False") == "True"
+                    self._usage_counter = {"get_online_features": 0}
 
-                    if os.path.exists(telemetry_filepath):
-                        with open(telemetry_filepath, "r") as f:
-                            self._telemetry_id = f.read()
+                    if os.path.exists(usage_filepath):
+                        with open(usage_filepath, "r") as f:
+                            self._usage_id = f.read()
                     else:
-                        self._telemetry_id = str(uuid.uuid4())
+                        self._usage_id = str(uuid.uuid4())
 
-                        with open(telemetry_filepath, "w") as f:
-                            f.write(self._telemetry_id)
+                        with open(usage_filepath, "w") as f:
+                            f.write(self._usage_id)
                         print(
                             "Feast is an open source project that collects anonymized error reporting and usage statistics. To opt out or learn"
                             " more see https://docs.feast.dev/reference/telemetry"
```

Comment:
would this link still work?

Reply:
oops, forgot to update this after documentation changes